server/helper/ImageUtils.js

```diff
@@ -3,7 +3,22 @@ const path = require("path");
 const fs = require("fs");
 const logger = require("../config/logger");
 
-const THUMBNAIL_POSTFIX = "-thumb";
+class ImageSize {
+  constructor(name, maxDimension) {
+    this.name = name;
+    this.maxDimension = maxDimension;
```

Comment:
maxDimension stimmt in dem Fall nicht, denn wir geben dem Browser ja die Breite des Bildes. Müsste also maxWidth sein.

Reply:
Wir resizen aber sowohl width als auch height. (siehe ImageUtils:69)
Dann müsste man das dort auch umbauen.